packages/table/src/common/classes.ts

```diff
@@ -0,0 +1,100 @@
+export const TABLE_DRAGGABLE = "bp-table-draggable";
+export const TABLE_DRAGGING = "bp-table-dragging";
+export const TABLE_EDITABLE_NAME = "bp-table-editable-name";
+export const TABLE_HEADER = "bp-table-header";
+export const TABLE_HEADER_ACTIVE = "bp-table-header-active";
+export const TABLE_HEADER_CONTENT = "bp-table-header-content";
+export const TABLE_HEADER_SELECTED = "bp-table-header-selected";
+export const TABLE_HORIZONTAL_CELL_DIVIDER = "bp-table-horizontal-cell-divider";
+export const TABLE_HORIZONTAL_GUIDE = "bp-table-horizontal-guide";
+export const TABLE_INTERACTION_BAR = "bp-table-interaction-bar";
+export const TABLE_LAST_IN_COLUMN = "bp-table-last-in-column";
+export const TABLE_LAST_IN_ROW = "bp-table-last-in-row";
+export const TABLE_MENU = "bp-table-menu";
+export const TABLE_NO_HORIZONTAL_SCROLL = "bp-table-no-horizontal-scroll";
+export const TABLE_NO_LAYOUT = "bp-table-no-layout";
+export const TABLE_NO_VERTICAL_SCROLL = "bp-table-no-vertical-scroll";
+export const TABLE_NULL = "bp-table-null";
+export const TABLE_OVERLAY = "bp-table-overlay";
+export const TABLE_OVERLAY_LAYER = "bp-table-overlay-layer";
+export const TABLE_POPOVER_WHITESPACE_NORMAL = "bp-table-popover-whitespace-normal";
+export const TABLE_POPOVER_WHITESPACE_PRE = "bp-table-popover-whitespace-pre";
+export const TABLE_REGION = "bp-table-region";
+export const TABLE_RESIZE_GUIDES = "bp-table-resize-guides";
+export const TABLE_RESIZE_HANDLE = "bp-table-resize-handle";
+export const TABLE_RESIZE_HANDLE_TARGET = "bp-table-resize-handle-target";
+export const TABLE_RESIZE_HORIZONTAL = "bp-table-resize-horizontal";
+export const TABLE_RESIZE_SENSOR = "bp-table-resize-sensor";
+export const TABLE_RESIZE_SENSOR_EXPAND = "bp-table-resize-sensor-expand";
+export const TABLE_RESIZE_SENSOR_SHRINK = "bp-table-resize-sensor-shrink";
+export const TABLE_RESIZE_VERTICAL = "bp-table-resize-vertical";
+export const TABLE_ROUNDED_LAYOUT = "bp-table-rounded-layout";
+export const TABLE_ROW_HEADERS = "bp-table-row-headers";
+export const TABLE_ROW_NAME = "bp-table-row-name";
+export const TABLE_ROW_NAME_TEXT = "bp-table-row-name-text";
+export const TABLE_SELECTION_ENABLED = "bp-table-selection-enabled";
+export const TABLE_SELECTION_REGION = "bp-table-selection-region";
+export const TABLE_TH_MENU = "bp-table-th-menu";
+export const TABLE_THEAD = "bp-table-thead";
+export const TABLE_TOP_CONTAINER = "bp-table-top-container";
+export const TABLE_TRUNCATED_FORMAT = "bp-table-truncated-format";
+export const TABLE_TRUNCATED_POPOVER = "bp-table-truncated-popover";
+export const TABLE_TRUNCATED_POPOVER_TARGET = "bp-table-truncated-popover-target";
+export const TABLE_TRUNCATED_TEXT = "bp-table-truncated-text";
+export const TABLE_TRUNCATED_VALUE = "bp-table-truncated-value";
+export const TABLE_VERTICAL_GUIDE = "bp-table-vertical-guide";
+
+/** Common code for row and column index class generator functions, since they're essentially the same */
```

Comment:
I think standard typedoc format is to have a newline after the `/**` like so:

```
/**
 * Common code for...
 */
```

Not sure if it'd change how it's rendered